eval/buffer.go

```diff
@@ -14,6 +17,19 @@ const (
 	BufferResponse
 )
 
+func (i BufferOption) GoString() string {
+	var result []string
+	for _, o := range []BufferOption{BufferRequest, BufferResponse} {
+		if (i & o) == o {
+			result = append(result, o.String())
+		}
+	}
+	if len(result) == 0 {
```

Comment:
This total `if {...}` block is obsolete.

Reply:
Without this block the result for
```go
var myOption BufferOption
fmt.Printf("%#v", myOption)
```

would be `""` Instead of `BufferNone`.